<!-- Philips Cam -->
<!-- TO SET BEFORE EXPERIMENT -->
<launch>
  <node name="philips_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
   <remap from="/philips_cam/image_raw" to="/usb/philips/cam_img"/>
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="camera_name" value="philips" />
  </node>   
  
<arg name="image_topic" default="/usb/philips/cam_img" />
<arg name="calibration_topic_name" default="/philips/calibration_topic"/>
<arg name="node_name" default = "philips_calibration"/>
<node pkg="nostop_kinect_sensor" type="script_calibration.sh" 
    args="$(arg node_name) $(arg image_topic) $(arg calibration_topic_name)"
    name="calibration_node" output="screen">
</node>

<arg name="camera_name" default="philips" />
  <node pkg="nostop_kinect_sensor" type="script_cam.sh" 
    args="$(arg camera_name) $(arg image_topic) $(arg calibration_topic_name)" 
    name="camera_node_philips" output="screen">
  </node>
  
  <node name="rqt_reconfigure_philips" pkg="rqt_reconfigure" type="rqt_reconfigure"/>
  
</launch>
